forge-gui-mobile/src/forge/adventure/util/Config.java

```diff
@@ -98,7 +106,7 @@ private Config() {
 
         //prefix = "forge-gui/res/adventure/Shandalar/";
         prefix = getPlanePath(settingsData.plane);
-        commonPrefix = getPlanePath("common");
+        commonPrefix = getPlanePath(commonDirectoryName);
```

Comment:
i still think common directory is not planepath but common resource folder for the planes, can the adventure directory have planes directory and put the amonkhet and shandalar there and just read there? I think this will eliminate the file namefilter since anything you put there are folder for planes.

Reply:
If it makes you feel better we can duplicate getPlanePath() with a different name... I don't think it makes sense for the common folder to be located anywhere other than with Amonkhet and Shandalar, since each is technically a collection of resources, not a plane.

Reply:
Updated so it's not using getPlanePath for that.

Reply:
I thought you want to organize this, so under adventure directory there is common folder and planes folder, and inside planes folder, amonkhet and shandalar reside. Doesn't that makes better?

Reply:
Usually a common folder shares the same home as the directories that override it. I've never seen an implementation that put the overrides all in a subdirectory.